.github/workflows/samples-java-client-jdk17.yaml

```diff
@@ -20,6 +22,7 @@ jobs:
           # clients
           - samples/client/petstore/java/resttemplate-jakarta
           - samples/client/petstore/java/webclient-jakarta
+          - samples/client/petstore/java/restclient-*/** 
```

Comment:
this needs to be the full path (no *)

Reply:
Oh yes, thanks :)

Reply:
@wing328 I fixed it, can you allow the new job run? 🙂